config.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL = postgres://yizthhwjzzmgrm:8d830f4e30e56aedc270b92fe7fdf98d083113b2169c99394e52d4a17ba4a808@ec2-50-19-222-129.compute-1.amazonaws.com:5432/d9bltlsuv2jqs
```

Comment:
You shouldn't have your config.env up on github, please add this to your .gitignore.

Reply:
Please remove this file from the cache.